code-challenges.test.js

```diff
@@ -49,4 +145,51 @@ const hand3 = [5, 5, 5, 5, 4]
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
+//PSUEDO CODE:
+// Name: fullHouse
+// input: array
+// output: true if the array includes 1 pair and 1 three of a kind. or false 
+// Process: 
+// - Use a conditional statement to first check if the array length is in face equal to 5, return false if it is not.
+//  - Create an empty object to store the count of each unique number in the array.
+//  - Use a for loop to iterate over each element in the array 
+// - For each element, the code checks if the object already has a property with that value as the key. If it does, it increments the count by 1. Otherwise it initalizes the count to 1. 
+// - The object.values() method extracts all the values from the object and store them in a new array. 
+// - Use the .includes() method to check if the array's values include both the numbers 2 and 3. This condition will return true or false, indicatiing a full house or not. 
```

Comment:
This is one of my favorite approaches to this challenge, well done! Your pseudo code explained it well! Nice job using the for of loop too!